packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts

```diff
@@ -156,7 +156,7 @@ export class OID4VP {
       context: this.session.context,
       domain: proofOptions.domain,
       challenge: proofOptions.challenge,
-      format: opts?.restrictToFormats ?? selectedVerifiableCredentials.definition.definition.format,
+      fallbackFormat: opts?.restrictToFormats ?? selectedVerifiableCredentials.definition.definition.format,
```

Comment:
This is no fallback format

Reply:
I'll rename it back, but it's the last option in 
```
 const formatOptions = format ?? presentationDefinition.format ?? args.fallbackFormat
```

Reply:
Yes which results in a format being used afterwards that is not a fallback in the end of course